src/components/Tile/Tile.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { MediaObject } from '../MediaObject';
+
+import Title from './components/Title';
+import TitleRow from './components/TitleRow';
+
+const Tile = (props) => {
+  const {
+    as,
+    board,
+    children,
+    className,
+    action,
+    figure,
+    title,
+    titleText,
+    ...rest
+  } = props;
+
+  const tileClasses = cx([
+    'slds-tile',
+    { 'slds-tile_board': board },
+    { 'slds-hint-parent': !!action },
+    className,
+  ]);
+
+  const tileContent = (
+    <React.Fragment>
```

Comment:
Do we have plans to finally upgrade our deps? To get rid of `<React.Fragment>`

Reply:
Good point, I will do that in a separate PR